fe/fe-core/src/main/java/org/apache/doris/catalog/FunctionRegistry.java

```diff
@@ -212,7 +212,7 @@ public FunctionBuilder findFunctionBuilder(String dbName, String name, List<?> a
         return candidateBuilders.get(0);
     }
 
-    private List<FunctionBuilder> findBuiltinFunctionBuilder(String name, List<?> arguments) {
+    public List<FunctionBuilder> findBuiltinFunctionBuilder(String name, List<?> arguments) {
```

Comment:
why change this?

Reply:
for `TypeCoercionUtils.processTimestampArithmetic`